src/shared/types/featureFlags.ts

```diff
@@ -1,9 +1,9 @@
 import { LDFlagValue } from 'launchdarkly-react-client-sdk';
 
 // These keys use the camelCase representation of the feature flag value
-// e.g. multi-informant-feature-flag in LaunchDarky becomes multiInformantFeatureFlag
+// e.g. enable-multi-informant in LaunchDarky becomes enableMultiInformant
 export const FeatureFlagsKeys = {
-  multiInformantFlag: 'multiInformantFeatureFlag',
+  multiInformantFlag: 'enableMultiInformant',
```

Comment:
Any reason not to align the key names with the values too?
```suggestion
  enableMultiInformant: 'enableMultiInformant',
```

Reply:
Mostly wanted to keep my change minimal...but you're right that we might as well have them be the same instead of the wild wild west. Updating!